SwiftWisdom/Core/StandardLibrary/Numbers/Int+Extensions.swift

```diff
@@ -24,3 +24,41 @@ extension IntegerType {
         (0..<self).forEach { _ in closure() }
     }
 }
+
+extension IntegerType {
+    public func ip_toMagnitudeString(decimalPlaces decimals: Double = 1) -> String {
```

Comment:
I think the signature should take an `Int`, I don't see anyone using `1.5` decimal places. I also think `decimalPlaces` should be used internally and externally. `decimals` reads a bit like the number of `.` instead of the number of places